docs/dev/metrics.md

```diff
@@ -111,3 +111,20 @@ mapi_instance_delete_failed{name=machine-3,namespace=openshift-machine-api,reaso
 ```
 
 [Demo](https://user-images.githubusercontent.com/32226600/87791648-e72b6900-c842-11ea-90b7-4967b0d06fb5.gif)
+
+## Metrics about MachineHealthCheck resources
+
+When using MachineHealthChecks, metrics are available from the `machine-api-controllers` Pod on the
+default metrics port(`8083`) for the `machine-healthcheck-controller` container. 
+
+The `mapi_machinehealthcheck_nodes_covered` metric describes the number of Nodes that are currently
+being monitored by `machine-healthcheck-controller`. The `name` label in this metric refers to the
+name of the MachineHealthCheck that is being reported.
```

Comment:
Do we need a namespace label? Is it possible to have MHCs in multiple namespaces in a cluster? Does it hurt having this extra dimension?

Reply:
i was curious about this, i _thought_ we only observed MHCs in the openshift-machine-api namespace, but i can certainly add the namespace label here as well. shouldn't hurt anything and we wouldn't export through telemetry.

Reply:
One of the things I was considering is in the future, if we host multiple control planes within a single cluster, would this then lead us to having potential namespace clashes, I think it would

Reply:
if i understand this code[0] correctly, i think we will always deploy the machine health check controller to watch the `openshift-machine-api` namespace. so we shouldn't run into any issues there, i imagine this would extend to the multi-control plane scenario as well since in those cases the prometheus servers would still be different instances.

all that said, i have a feeling adding the namespace for the metric is a good idea since technically the mhc controller can be made to watch all namespaces. i'm going to update this and the enhancement, we will still refrain from publishing the name and namespace through telemetry.